app/components/gallery/PictureItem.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <a href="#" @click.prevent="shadowbox = !shadowbox">
+    <div class="photo aspect-ratio-1/1">
+      <img class="image-fit" :src="photo.full_image_url" :srcset="photo.medium_srcset" :alt="photo.title">
+      <div class="description">
+        {{ photo.title }}
+      </div>
+    </div>
+    <Modal :media="photo.full_image_url" :title="photo.title" :visibility="shadowbox" />
+  </a>
+</template>
+
+<script>
+export default {
+  props: {
+    photo: {
+      type: Object,
+      required: true
+    }
+  },
+  data () {
```

Comment:
is `data` here a required function name if not it might be renamed to something reflecting the return value